Livecheckables/genders.rb

```diff
@@ -0,0 +1,6 @@
+class Genders
+  livecheck do
+    url "https://github.com/chaos/genders/releases/latest"
+    regex(%r{href=.*?/tag/genders-(\d+(?:-\d+)+)["']}i)
```

Comment:
```suggestion
    regex(%r{href=.*?/tag/genders-v?(\d+(?:-\d+)+)["']}i)
```

I still like to have the optional `v?`, just in case.